Random Users/Networking/UserClient.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  UserClient.swift
+//  Random Users
+//
+//  Created by ronald huston jr on 9/12/20.
+//  Copyright © 2020 Erica Sadun. All rights reserved.
+//
+
+import Foundation
+
+class UserClient {
+    
+    let baseURL = URL(string: "https://randomuser.me/api/?format=json&inc=name,email,phone,picture&results=1000")!
+    
+    func fetchUsers(completion: @escaping ([User]?, Error?) -> Void) {
+        URLSession.shared.dataTask(with: baseURL) { (data, _, error) in
```

Comment:
You are not handling the potential incorrect response code sent back from the api 
```suggestion
        URLSession.shared.dataTask(with: baseURL) { (data, response, error) in
```